index.html

```diff
@@ -85,14 +85,26 @@ <h4 class="variableTitle">Variables</h4>
             <div class="vizCol">
                 <h4 class="graphsTitle">Graphs</h4>
                 <div class="dataViz">
-                <canvas id="firstGraph"></canvas>
-                <canvas id="secondGraph"></canvas>
+                    <div class="canvas-outer-bar">
+                        <div class="canvas-inner">
+                            <canvas id="firstGraph"></canvas>
+                        </div>
+                    </div>
+                    <div class="canvas-outer-radar">
```

Comment:
This canvas wrapper is a bit smaller. I didn't add a wrapper to the third canvas/graph. If you it suits you, you can add a wrapper to it or remove this wrapper if it doesn't work for you.